### PR TITLE
composer: PHP 7 added

### PR DESCRIPTION
**Pull request by TomasVotruba**
Also drop default value for notifications.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,17 +5,17 @@ php:
...
+    fast_failure: true
 
 install:
-  - composer install --prefer-source --no-interaction
+    - composer install --prefer-source --no-interaction
 
 script:
-  - phpunit
-
-notifications:
```
Comment (mmoreram) [inline .travis.yml]: why do you remove this?

Comment (TomasVotruba) [inline .travis.yml]: Sorry, I didn't realize it disables emails.

Comment (TomasVotruba) [inline .travis.yml]: Fixed :)

**Comment by TomasVotruba** [comment]
Btw, could you also add this package to Travis? It looks like it's not triggered.

**Comment by mmoreram** [comment]
Done :D Thanks for that warning! :o

**Comment by TomasVotruba** [comment]
Thanks!

**Pull request merged**